pyfar/dsp/filter/fractional_octaves.py

```diff
@@ -329,6 +381,65 @@ def _coefficients_fractional_octave_bands(
     return sos
 
 
+def _check_fractional_octave_filter_tolerance(
```

Comment:
would it make sence to make this method also public?
an other approach could be to make this method public and return true or false. Then we dont need to change ``fractional_octave_bands`` but just add an example at using this method to check for valid filter tolerances